webpack.config.js

```diff
@@ -27,21 +27,22 @@ const ComponentsConfig = {
     module: {
         rules: [
             {
-                test: /\.scss$/i,
+                test: /\.css$/i,
+                exclude: /\.module\.scss$/i,
                 sideEffects: true,
                 use: [
                     'style-loader',
-                    // Translates CSS into CommonJS
+                    'css-loader',
                     {
-                        loader: 'css-loader',
+                        loader: 'sass-loader',
```

Comment:
I'm not quite sure about this, but why would we need a sass loader here to load css that are already compiled from sass ?

Reply:
I think it was left over from a previous change but I tested it out with just css-loader and everything seems to be fine 👍🏾